### ISSUE TITLE
Detect missing AutomationProperties

### ISSUE THREAD

**Issue by mrlacey** [mrlacey/Rapid-XAML-Toolkit#242]
To aid accessibility in the final app, there are probably things that can be detected at design time as possible addressable issues. Initially thinking about the narrator, but other accessibility issues considered.

- [ ] Buttons displaying an image but with no alternative text.
- [ ] Image without AP.Description (alt text)
- [ ] Lack of access key?
- [ ] Missing HeadingLevel?
- [ ] Form elements without `IsRequiredForForm`
- [ ] form elements with separate labels that don't set `LabeledBy`
- [ ] Lists without `SizeOfSet`

open to other ideas....
